events/amld-workshop/README.md

```diff
@@ -0,0 +1,72 @@
+- 11:00 – 11:45 – EV Range Prediction
+- 12:00 – 12:45 – Tracking Covid
+- 12:45 – 14:00 - LUNCH
+- 14:00 – 14:45 – Covid & Amazon Product’s Reviews
+- 15:00 – 16:00 – Finalize Practical Tutorias + Wrap Up
+## EV Range Prediction
+We will use a simple linear regression model to calculate the range that we can drive under various meteorological conditions and speed.
+We will create a data driven application powered by a data science project. We will demonstrate how we can sustainably move our POC data science projects into production-ready products.
+### [Launch Workshop 1](https://github.com/versatile-data-kit-amld/linear-regression-example-unsolved)
+## Tracking Covid
+We are going to build a data application that would track Covid cases and deaths in a randomly selected set of European countries. We will show how to obtain data from multiple sources, ingest such sets of data, clean them, and combine them to create one data driven product, which is being updated regularly.
+### [Launch Workshop 2](https://github.com/versatile-data-kit-amld/tracking-covid-cases-deaths-example-unsolved)
+## Covid vs No Scent Complaints
+Ispired by a Twitter post, we will investigate whether there is a connection between Covid cases in the US and complaints raised for one Amazon product.
+We will go through the whole data science workflow from obtaining the data, cleaning it, doing a correlation analysis, and building a data-driven application that is being updated regularly. And this would be achieved with minimal additional effort than running the analysis in POC.
+### [Launch Workshop 3](https://github.com/versatile-data-kit-amld/correlation-example-unsolved)
+
+## Feedback
+Please share your feedback : https://bit.ly/vdk-amld
+
+Give as a star if you've liked the [project](https://github.com/vmware/versatile-data-kit).
+
+## Where to Find Us
+- [YouTube](https://www.youtube.com/channel/UCasf2Q7X8nF7S4VEmcTHJ0Q/about)
+- [Twitter](https://twitter.com/vdkproject)
+- [GitHub](https://github.com/vmware/versatile-data-kit)
+- Relevant Articles
+    - [An Overview of Versatile Data Kit](https://towardsdatascience.com/an-overview-of-versatile-data-kit-a812cfb26de7)
+    - [From Raw Data to a Cleaned Database: A Deep Dive into Versatile Data Kit](https://towardsdatascience.com/from-raw-data-to-a-cleaned-database-a-deep-dive-into-versatile-data-kit-ab5fd992a02e)
+## Workshop Solutions
+- [EV Range Prediction (Solved)](https://github.com/versatile-data-kit-amld/linear-regression-example-solved)
+- [Tracking Covid (Solved)](https://github.com/versatile-data-kit-amld/tracking-covid-cases-deaths-example-solved)
+- [Covid vs. No Scent Complaints (Solved)](https://github.com/versatile-data-kit-amld/correlation-example-solved)
+
+
+## Run workshop on local machine
+
+The workshops run in mybinder.org environment by default. But if you want to run it locally follow [instructions here](https://github.com/versatile-data-kit-amld/workshop/blob/main/backup/local_instructions.md)
```

Comment:
Where is this link supposed to point to?

Reply:
Just committed a change with updated link.